examples/transfer-learning/plot_classify_mdwm.py

```diff
@@ -0,0 +1,288 @@
+"""
+=====================================================================
+ Apply MDWM transfer learning to classify a set of matrices
+=====================================================================
+
+Applying MDWM to learn from source matrices and classify target matrices.
```

Comment:
the example does not explain that MDWM means Minimum Distance to Weighted Mean. The header of the example should explain a few things.